src/test/java/net/openhft/performance/tests/network/BufferSizeTest.java

```diff
@@ -135,8 +131,9 @@ private void sendAndReceive(String expectedMessage, int tcpBufferSize) throws IO
                 totalRead += read;
                 ++count;
             }
-            if (count > 1)
-               // System.out.println("count=" + count);
+            if (count > 1) {
+                // System.out.println("count=" + count);
+            }
```

Comment:
This is why we should always use brackets even if there's only a single line :wink: (notice the test is no longer disabled)